### PR TITLE
docs: update platform-related docs pages

### PR DESCRIPTION
**Pull request by huanjani**
Update docs with platform info and move changes out of outdated file.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/docs/include/common-svc-fields.en.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Amount of memory in MiB used by the task. See the [Amazon ECS docs](https://docs
 <div class="separator"></div>
 
 <a id="platform" href="#platform" class="field">`platform`</a> <span class="type">String or Map</span>  
-Non-default operating system and architecture (formatted as `[os]/[arch]`) to pass with `docker build --platform`.
+Operating system and architecture (formatted as `[os]/[arch]`) to pass with `docker build --platform`. For example, `linux/arm64` or `windows/x86_64`. The default is `linux/x86/64`.
```
Comment (efekarakus) [inline site/content/docs/include/common-svc-fields.en.md]: typo 🙈 
```suggestion
Operating system and architecture (formatted as `[os]/[arch]`) to pass with `docker build --platform`. For example, `linux/arm64` or `windows/x86_64`. The default is `linux/x86_64`.
```

File: site/content/docs/include/platform.en.md [hunk 2]
```diff
@@ -1,4 +1,15 @@
 <div class="separator"></div>
 
-<a id="platform" href="#platform" class="field">`platform`</a> <span class="type">String</span>  
-Operating system and architecture (formatted as `[os]/[arch]`) to pass with `docker build --platform`.
\ No newline at end of file
+<a id="platform" href="#platform" class="field">`platform`</a> <span class="type">String</span>
+Operating system and architecture (formatted as `[os]/[arch]`) to pass with `docker build --platform`. For example, `linux/arm64` or `windows/x86_64`. The default is `linux/x86/64`.
```
Comment (efekarakus) [inline site/content/docs/include/platform.en.md]:
```suggestion
Operating system and architecture (formatted as `[os]/[arch]`) to pass with `docker build --platform`. For example, `linux/arm64` or `windows/x86_64`. The default is `linux/x86_64`.
```

File: site/content/docs/manifest/scheduled-job.en.md [hunk 3]
```diff
@@ -102,9 +102,19 @@ Amount of memory in MiB used by the task. See the [Amazon ECS docs](https://docs
 
 <div class="separator"></div>
 
-<a id="platform" href="#platform" class="field">`platform`</a> <span class="type">String</span>  
-Operating system and architecture (formatted as `[os]/[arch]`) to pass with `docker build --platform`.
+<a id="platform" href="#platform" class="field">`platform`</a> <span class="type">String</span>
+Operating system and architecture (formatted as `[os]/[arch]`) to pass with `docker build --platform`. For example, `linux/arm64` or `windows/x86_64`. The default is `linux/x86/64`.
```
Comment (efekarakus) [inline site/content/docs/manifest/scheduled-job.en.md]:
```suggestion
Operating system and architecture (formatted as `[os]/[arch]`) to pass with `docker build --platform`. For example, `linux/arm64` or `windows/x86_64`. The default is `linux/x86_64`.
```

**Review by efekarakus — APPROVED** [review]

File: site/content/docs/include/common-svc-fields.en.md [hunk 4]
```diff
@@ -34,7 +34,7 @@ Amount of memory in MiB used by the task. See the [Amazon ECS docs](https://docs
 <div class="separator"></div>
 
 <a id="platform" href="#platform" class="field">`platform`</a> <span class="type">String or Map</span>  
-Non-default operating system and architecture (formatted as `[os]/[arch]`) to pass with `docker build --platform`.
+Operating system and architecture (formatted as `[os]/[arch]`) to pass with `docker build --platform`. For example, `linux/arm64` or `windows/x86_64`. The default is `linux/x86_64`.
```
Comment (Lou1415926) [inline site/content/docs/include/common-svc-fields.en.md:37]: Looks like this file isn't used any where anymore. We should probably just remove it some day after we make sure it's not referenced at all.

**Review by Lou1415926 — APPROVED** [review]

**Pull request merged**